android-iconify-fontawesome/build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.0'
+        classpath 'com.android.tools.build:gradle:2.1.0-rc1'
```

Comment:
I'm a little nervous about requiring something that's still in the RC stage. Any idea when this is supposed to be final? Can we just still for a week?